sites/public/pages/index.tsx

```diff
@@ -35,18 +46,97 @@ export default function Home() {
   )
   interface LatestListingLinkProps {
     name: string
-    address: string
-    availability: string
+    address: Address
+    imageUrl: string
+    availability?: string
   }
 
+  const linearGradient = "linear-gradient(to bottom, rgba(255, 255, 255, 0), rgba(24, 37, 42, .8))"
   const LatestListingsLink = (props: LatestListingLinkProps) => (
-    <a className={styles["latest-listing"]} href="/listings">
+    <a
+      className={styles["latest-listing"]}
+      href="/listings"
+      style={{
+        backgroundImage: `${linearGradient}, url(${props.imageUrl}`,
+      }}
+    >
       <h3 className={styles["latest-listing__name"]}>{props.name}</h3>
-      <p className={styles["latest-listing__address"]}>{props.address}</p>
-      <div className={styles["latest-listing__availability"]}>{props.availability}</div>
+      <p className={styles["latest-listing__address"]}>
+        <OneLineAddress address={getGenericAddress(props.address)} />
+      </p>
+      {props.availability && (
+        <div className={styles["latest-listing__availability"]}>{props.availability}</div>
+      )}
     </a>
   )
 
+  /**
+   * Get the listing with the latest 'updatedAt' field and format it
+   *
+   * TODO(#672): Translate last updated string
+   * @param listings
+   * @returns
+   */
+  const getLastUpdatedString = (listings: Array<Listing>) => {
+    // Get the latest updateAt date and format it as localized 'MM/D/YYYY'
+    const latestDate = moment
+      .max(
+        listings.map((listing) => {
+          return moment(listing.updatedAt)
+        })
+      )
+      .format("l")
+    return `Last updated ${latestDate}`
+  }
+
+  /**
+   * Convert the number of bedrooms to a human readable string
+   *
+   * TODO(#672): Translate unitBedrooms string
+   * @param numBedrooms
+   * @param plural
+   * @returns
+   */
+  const unitBedroomsToString = (numBedrooms: number, plural: boolean) => {
```

Comment:
I thought there was already strings in the translated json for this?

Reply:
Do you mean in locales/general.json? I don't see anything there other than the "bedroomOptions" for the listing filters which seems to be a little different. Am I missing it?

I did see there was a name string in the UnitType, but chose not to use that because a) at least in the seed data we have it seemed to not be consistent and to use some weird abbreviations? and b) those can't be translated

Reply:
Ah, ok, I just thought there was something already like this, but maybe not. The strings in the database will be machine translated via Google Translate, as soon as I get around to changing the translation service.